config/options.go

```diff
@@ -242,9 +242,9 @@ type Options struct {
 	ForwardAuthURLString string   `mapstructure:"forward_auth_url" yaml:"forward_auth_url,omitempty"`
 	ForwardAuthURL       *url.URL `yaml:",omitempty"`
 
-	// DataBrokerURL is the routable destination of the databroker service's gRPC endpiont.
-	DataBrokerURLString string   `mapstructure:"databroker_service_url" yaml:"databroker_service_url,omitempty"`
-	DataBrokerURL       *url.URL `yaml:",omitempty"`
+	// DataBrokerURLString is the routable destination of the databroker service's gRPC endpoint.
+	DataBrokerURLString  string   `mapstructure:"databroker_service_url" yaml:"databroker_service_url,omitempty"`
+	DataBrokerURLStrings []string `mapstructure:"databroker_service_urls" yaml:"databroker_service_urls,omitempty"`
```

Comment:
maybe handle them similar to `To` via StringSlice for consistency?

Reply:
mapstructure handles string slices natively. For `to` I wrote a custom type because we had to maintain compatibility with the previous format. Unfortunately this requires a very complex mapstructure decoding hook to handle properly.

I can do the same for `databroker_service_url` though I'd rather avoid it. If you disagree I can proceed with making `databroker_service_url` a `StringSlice` and handle single strings or lists of strings.